sycl/include/CL/sycl/intel/fpga_device_selector.hpp

```diff
@@ -32,9 +32,10 @@ class platform_selector : public device_selector {
   }
 };
 
-const std::string EMULATION_PLATFORM_NAME =
+static constexpr char const *EMULATION_PLATFORM_NAME =
     "Intel(R) FPGA Emulation Platform for OpenCL(TM)";
-const std::string HARDWARE_PLATFORM_NAME = "Intel(R) FPGA SDK for OpenCL(TM)";
+static constexpr char const *HARDWARE_PLATFORM_NAME =
```

Comment:
'constexpr + const' seems too much to me )) , but probably Ok.

Reply:
`static constexpr auto HARDWARE_PLATFORM_NAME =` is simpler